package.json

```diff
@@ -428,6 +452,10 @@
                 {
                     "command": "hubspot.remoteFs.invalidateCache",
                     "when": "false"
+                },
+                {
+                    "command": "hubspot.remoteFs.startWatch",
+                    "when": "false"
                 }
```

Comment:
`hubspot.remoteFs.watch` and `hubspot.remoteFs.endWatch` are UI hooks, `hubspot.remoteFs.startWatch` is an internal command. It might be worth finding a naming scheme that makes this obvious for maintainability's sake